libs/shared/src/consts/providers/credentials/provider-credentials.ts

```diff
@@ -450,6 +450,24 @@ export const expoConfig: IConfigCredentials[] = [
   ...pushConfigBase,
 ];
 
+export const pushApiConfig: IConfigCredentials[] = [
+  {
+    key: CredentialsKeyEnum.PrivateKey,
```

Comment:
I wonder if we can reuse SecretKey instead of creating a new one, the name won't need to be the same just to have the same purpose.

Reply:
I thought about that too. At first I had it the same, but I switched it since I was already adding a new public key as well. I figured in the future there might be another use case for a private key as well. It's up to your team formally. It would be little effort to switch it out.